helm-chart/README.rst

```diff
@@ -64,6 +64,46 @@ in the `values changelog file <https://github.com/SwissDataScienceCenter/renku/b
 For upgrades that require some steps other than modifying the values files to be
 executed, we add some instructions here.
 
+Upgrading to 0.8.0
+******************
+We bump the PostgreSQL version from 9.6 to 11 and the GitLab major version from 11 to 13.
+It is important to first perform the postgresql upgrade, then upgrade to the ``0.8.0`` chart version
+while keeping the GitLab version fixed, and finally upgrade the GitLab version.
+
+1. Upgrading postgresql
++++++++++++++++++++++++
+If PostgreSQL was deployed as part of Renku, please follow [these instructions](https://github.com/SwissDataScienceCenter/renku/tree/master/helm-chart/utils/postgres_migrations/version_upgrades/README.md)
+for the postgresql upgrade.
+
+2. Bump the chart version
++++++++++++++++++++++++++
+Now it's time to upgrade to the ``0.8.0`` version of the Renku chart. Before doing this, make sure
+to pin the GitLab version by setting ``gitlab.image.tag`` in your values file. If you had not pinned
+this version explicitly before, pin it to ``11.9.11-ce.0`` which is the default version set in the Renku
+chart prior to the upgrade. Otherwise you can leave it at the previously pinned version. Then deploy the
+new chart version through ``helm upgrade ... --version 0.8.0 ...``.
+
+3. Upgrade GitLab
++++++++++++++++++
+Please read the `GitLab documentation on this topic <https://docs.gitlab.com/ce/update>`_ before proceeding.
+Following the `recommended upgrade paths<https://docs.gitlab.com/ce/update/#upgrade-paths>`_) and assuming
+your GitLab instance is at version ``11.9.11``, this means that your upgrade path will be
+``11.11.8 -> 12.0.12 -> 12.1.17 -> 12.10.14 -> 13.0.14 -> 13.1.11 -> 13.10.4``. The corresponding
+image tags are:
+
+- 11.11.8-ce.0
+- 12.0.12-ce.0
+- 12.1.17-ce.0
+- 12.10.14-ce.0
+- 13.0.14-ce.0
+- 13.1.11-ce.0
+- 13.10.4-ce.0 (default in the Renku ``0.8.0`` helm chart)
+
+For each step, set the corresponding tag in your values file at ``gitlab.image.tag``, redeploy through
+helm and wait for the gitlab pod to be recreated and all migrations to finish. Repeat this procedure until
```

Comment:
To me, it's faster to just replace the image by editing the deployment. Especially if there are a lot of upgrades to be done. Then I'd use the `helm upgrade` only to 'persist' the latest version.

Reply:
I think that's a viable option for people with more experience, but for the general instructions I'd prefer to keep it at the level of the chart values.

Reply:
Yes, I guess it makes sense. We should have pinned newer GitLab versions in our default values file a long time ago. At least we're soon going to change this.